src/ui/flutter_app/lib/l10n/intl_en.arb

```diff
@@ -336,9 +392,15 @@
     "@about": {
         "description": "About"
     },
-    "version": "Version",
+    "version": "Version: {version_number}",
     "@version": {
-        "description": "Version"
+        "description": "Version",
+        "placeholders": {
+            "version_number": {
+                "description": "the current Sanmill version",
```

Comment:
The.

The first letter needs to be capitalized.

Reply:
Code comments can be arbitrary, this file is exposed to the translator, so it needs to be more formal.